modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,424 @@
+<?php
+/**
+ * This script sends HTTP request to every enpoints of the api module and look at the
+ * response content, status code and headers where it applies.
+ *
+ * PHP Version 7
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Integration
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+namespace LORIS\api\Test;
+require_once __DIR__ .
+"/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+require './vendor/autoload.php';
+
+use Laminas\Diactoros\Request;
+use GuzzleHttp\Client;
+
+/**
+ * PHPUnit class for API testsuite
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+class ProjectsTest extends TestCase
+{
+
+    private $_client;
+    private $_headers;
+    private $_base_uri;
+
+    /**
+     * Used to log in
+     *
+     * @return void
+     */
+    private function _login()
+    {
+        $this->_base_uri = $this->url . '/api/v0.0.3/';
+        $this->_client   = new \GuzzleHttp\Client(['base_uri' => $this->_base_uri]);
+        $response        = $this->_client->request('POST', $this->_base_uri . '/login', ['json' => ['username' => UnitTester, 'password' => $this->validPassword]]);
+        $token           = json_decode($response->getBody()->getContents())->token ?? null;
```

Comment:
You could add a test after this line to check if $token is null.
phpunit provides assertions, it could be used here.